content/en/security_platform/guide/automate-the-remediation-of-detected-threats.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Automate the Remediation of Detected Threats with Webhooks
+kind: guide
+further_reading:
+- link: "/security_platform/explorer/"
+  tag: "Documentation"
+  text: "Start investigating signals in the Signals Explorer"
+
+---
+
+## Overview
+
+[Security Monitoring][1] allows you to set Detection Rules that trigger auto-remediation workflows. With Datadog’s [webhook integration][2], set up webhooks to deliver payloads to the services you want to automate whenever a [Detection Rule][3] is broken. Every webhook payload contains information about the triggering event and a custom message that can be used to initiate services downstream. Automate commands for any service that has a webhook URL.
+
+Choose a security scenario below to begin automating remediation.
+
+## Delete misconfigured security groups
+
+In a cloud environment, it’s important to delete a misconfigured resource as soon as it is created. In this scenario, you can configure a [webhook integration][2] to send a [webhook][2] to your cloud provider’s API management service.
+
+{{< img src="security_platform/security_monitoring/guide/automate-the-remediation-of-detected-threats/automation-diagram.png" alt="A diagram for a webhook sent to a cloud provider's API" >}}
+
+Once configured, if an AWS user creates a poorly configured resource (for example, an overly permissive security group, user role, etc.) within your AWS environment, Datadog Log Management will ingest the related log, which triggers a security group–based Detection Rule. This process will automatically send the webhook’s JSON payload to the designated AWS API Gateway URL, which in turn activates a AWS Lambda function that automatically deletes the offending resource.
+
+## Ban a suspicious IP address
+
+A sign-in from an unrecognized IP address might represent an attacker manipulating a trusted user’s credentials, with which they can then access your data and gain persistence in your environment.
+
+To combat this type of attack, you can use the [new term detection method][4], which analyzes your account’s historical data over a chosen period of time and alerts on previously unseen values in your cloud logs.
+
+First, set up a [new term–based Detection Rule][5].
+
+{{< img src="security_platform/security_monitoring/guide/automate-the-remediation-of-detected-threats/new-term-rule.png" alt="A new term-based detection rule" >}}
+
+Then, set up a [webhook][2] that will send a payload to your cloud’s identity and access management (IAM) service to ban the unknown IP when this rule is triggered.
+
+{{< img src="security_platform/security_monitoring/guide/automate-the-remediation-of-detected-threats/webhook-ip.png" alt="A new webhook that bans an unknown IP address" >}}
+
+The following example illustrates what the relevant webhook payload could look like when a security signal is produced by Datadog:
+
+{{< code-block lang="json" filename="webhook-payload.json" >}}
```

Comment:
@jmassey-datadog Hey there! Need verification / review here that these code blocks are good to go / any add't recommendations you may have for the guide.